src/auth/guards/auth.guard.ts

```
@@ -0,0 +1,50 @@
+  ) {
+    this.authTypeGuardMap = {
+      [authTypes.Bearer]: this.accessTokenGuard,
+      [authTypes.None]: { canActivate: () => true },
+    };
+  }
+
+  @ApiBearerAuth()
+  @ApiUnauthorizedResponse({ description: 'Unauthorized' })
+  async canActivate(context: ExecutionContext): Promise<boolean> {
+    const authTypes =
+      this.reflector.getAllAndOverride(AUTH_TYPE_KEY, [
```

ESLint

[error] Delete `⏎·····`

(prettier/prettier)